briskedu/brisk-learning
# Make error handling and permissions more robust

Summary:
Making the following changes:
1. No longer requesting classroom permissions which seem unnecessary
2. Making login window large enough to show all permissions
3. Showing error if user doesn't check permission boxes
4. Making all error handling handled by catching exceptions

Test Plan:
1. Login but don't check both boxes and confirm I get message about it when I see Google Auth login again
2. Reset and confirm if I only check revisions box, it works
3. Confirm once I check for AI, it shows login and asks for permissions again
<img width="311" alt="scopes" src="https://user-images.githubusercontent.com/15060005/226420841-322369b3-3af3-444f-8ae4-72f057ebfa27.png">
